### PR TITLE
fix(nav): updates to horizontal subnav

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/6299

### CODE REVIEW

File: src/patternfly/components/Page/page.scss [hunk 1]
```diff
@@ -151,14 +151,11 @@ $pf-page-v5--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
...
   // Main section horizontal subnav
-  --#{$page}__main-subnav--BackgroundColor: var(--pf-t--global--background--color--secondary--default);
-  --#{$page}__main-subnav--BorderTopWidth: var(--pf-t--global--border--width--divider--default);
-  --#{$page}__main-subnav--BorderTopColor: var(--pf-t--global--border--color--default);
-  --#{$page}__main-subnav--BorderLeftWidth: 0;
-  --#{$page}__main-subnav--BorderLeftColor: var(--pf-t--global--border--color--default);
-  --#{$page}__sidebar--main__main-subnav--BorderLeftWidth: var(--pf-t--global--border--width--divider--default);
-  --#{$page}__sidebar--m-collapsed--main__main-subnav--BorderLeftWidth: 0;
-
+  --#{$page}__main-subnav--BackgroundColor: var(--pf-t--global--background--color--primary--default);;
```
Comment (srambach) [inline src/patternfly/components/Page/page.scss:154]: semisemi :)

**Comment by tlabaj** [comment]
Should the button have the `pf-m-disabled` modifier?  It does in react hen `isDisabled` is applied.

image

**Comment by mcoker** [comment]
@tlabaj only when the button isn't a `<button>` element. For `<button>` elements, we can just style via the `disabled` attribute (`:disabled { ... }`). But when the element is an `<a>` (https://www.patternfly.org/components/button#links-as-buttons) or `<span>` (https://www.patternfly.org/components/button#inline-link-as-span), the `disabled` attribute isn't valid, so we style it to look disabled via the class `pf-m-disabled`

**Comment by srambach** [comment]
One other comment - the focus rings on the scroll buttons are a bit cut off and weird, but that's definitely fine for a follow up issue.

**Comment by mcoker** [comment]
Screenshots of a nav demo with horizontal subnav with scrollbars

Screenshot 2024-03-05 at 6 26 53 PM
Screenshot 2024-03-05 at 6 26 43 PM

**Review by srambach — APPROVED** [review]
Looks fabulous! Love the focus 💍

**Pull request merged**